### PR TITLE
RHBPMS-4959 - Tasks: GET request returns SessionNotFoundException whe…

### PR DESCRIPTION
**Pull request by mswiderski**
…n task belongs to aborted process instance

### CODE REVIEW

File: jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/UserTaskServiceImplTest.java [hunk 1]
```diff
@@ -30,6 +30,7 @@
 import org.jbpm.kie.services.impl.KModuleDeploymentUnit;
 import org.jbpm.kie.test.util.AbstractKieServicesBaseTest;
 import org.jbpm.services.api.ProcessInstanceNotFoundException;
+import org.jbpm.services.api.TaskNotFoundException;
```
Comment (MarianMacik) [inline jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/UserTaskServiceImplTest.java]: I would say that this import is unused.

Comment (mswiderski) [reply jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/UserTaskServiceImplTest.java]: fixed

**Comment by tsurdilo** [comment]
Jenkins retest please

**Comment by mswiderski** [comment]
@tsurdilo these failures seems to be related to my changes, will work on the,

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by mswiderski** [comment]
Jenkins execute full downstream build

**Comment by mswiderski** [comment]
test failure not related and the main purpose of full downstream build was to verify droolsjbpm-integration modules.

**Pull request merged**